catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Node.js

```diff
@@ -247,6 +247,12 @@ define([
             }
             return undefined;
         },
+        getRemoteNodes: function(){
+            return this.models.filter(function(model){
+                var transValues = model.get('TransientValues');
+                return !transValues || (!transValues['registry-identity-node'] && !transValues['registry-local-node']);
```

Comment:
Couldn't we just check the one 'registry-local-node' property since the identity node will have it as well?